Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -501,7 +501,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        return empty($changed) ? 0 : 1;
+        return $resolver->isDryRun() || empty($changed) ? 0 : 2;
```

Comment:
the discussion in #1041 was asking exactly the opposite of what this PR is doing.

It should return a 0 exit code whne things have been fixed successfully. so the non-0 exit code should be returned for cases where there are changes *and we are in dry-run mode*. The proper condition should be ``!$resolver->isDryRun()``